specification/applicationinsights/resource-manager/microsoft.insights/stable/2015-05-01/analyticsItems_API.json

```diff
@@ -0,0 +1,402 @@
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ItemScopePathParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ItemIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ItemNameParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The item has been successfully removed from the Application Insights component"
+                    }
+                },
+                "x-ms-examples": {
+                    "AnalyticsItemDelete": {
+                        "$ref": "./examples/AnalyticsItemDelete.json"
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "ApplicationInsightsComponentAnalyticsItem": {
+            "description": "Properties that define an Analytics item that is associated to an Application Insights component.",
+            "type": "object",
+            "properties": {
+                "Id": {
+                    "type": "string",
+                    "description": "Internally assigned unique id of the item definition."
+                },
+                "Name": {
+                    "type": "string",
+                    "description": "The user-defined name of the item."
+                },
+                "Content": {
+                    "type": "string",
+                    "description": "The content of this item"
+                },
+                "Version": {
+                    "type": "string",
+                    "description": "This instance's version of the data model. This can change as new features are added.",
+                    "readOnly": true
+                },
+                "Scope": {
+                    "description": "Enum indicating if this item definition is owned by a specific user or is shared between all users with access to the Application Insights component.",
+                    "type": "string",
+                    "enum": [
+                        "shared",
+                        "user"
+                    ],
+                    "x-ms-enum": {
+                        "name": "ItemScope",
+                        "modelAsString": true
+                    }
+                },
+                "Type": {
+                    "description": "Enum indicating the type of the Analytics item.",
+                    "type": "string",
+                    "enum": [
+                        "Query",
```

Comment:
Your example `AnalyticsItemList`, `AnalyticsItemGet` and `AnalyticsItemPut` disagree with spelling here (seem to be lowercase) - please adjust either examples or Swagger, whichever is wrong 🙂